services/efs-guardian/src/simcore_service_efs_guardian/cli.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+import typer
+from settings_library.utils_cli import create_settings_command
+
+from ._meta import APP_NAME
+from .core.settings import ApplicationSettings
+
+log = logging.getLogger(__name__)
+
+# NOTE: 'main' variable is referred in the setup's entrypoint!
+main = typer.Typer(name=APP_NAME)
+
+main.command()(create_settings_command(settings_cls=ApplicationSettings, logger=log))
+
+
+@main.command()
+def run():
+    """Runs application"""
+    typer.secho("Sorry, this entrypoint is intentionally disabled. Use instead")
```

Comment:
Do you have to add the command? If not needed I would simply leave it out.

Reply:
Yes, it's a skeleton then we have the same in each service. Potentially, it might be used in the future.